lib/request.coffee

```diff
@@ -172,6 +174,8 @@ module.exports = getRequest = ({
 	# .get('body')
 	###
 	exports.send = (options = {}) ->
+		{ responseFormat } = options
+		delete options.responseFormat
```

Comment:
It's not safe to mutate `options` here.

`options` is attached to the ResinRequestError below, if either the request or response fails. Downstream, the UI catches these errors, and retries requests by calling `.send` using the options from the error. With this change, if the UI ever retries a request, it'll drop the `responseFormat` for the retry, and create some fun bugs.